shared/helpers/abi.js

```diff
@@ -0,0 +1,26 @@
+const abi = require('web3-eth-abi')
+
+function stringifyParams(params) {
+  return params.map(p => {
+    if (Array.isArray(p)) {
+      return stringifyParams(p)
+    }
+    else {
+      if (typeof(p) === 'boolean') {
+        return p
+      }
+      else {
+        return `${p}`
+      }
+    }
+  })
+}
+
+function encodeFunctionCall(funcSignature, types, params) {
```

Comment:
Can we not just use `web3-eth-abi`'s [`encodeFunctionCall()`](https://github.com/ethereum/web3.js/blob/1.x/packages/web3-eth-abi/src/index.js#L200) directly?